src/plugins/recordTypes/dutyofcare/fields.js

```diff
@@ -87,15 +87,11 @@ export default (configContext) => {
             },
           },
         },
-        dutyOfCareTitle: {
+        title: {
```

Comment:
My comment on the redundancy of "Duty of care title" was really just about the label, since people see it and I want to keep the UI from being too busy. I don't care all that much about the field name, so as far as I'm concerned you didn't need to change it. But it's fine that you did. Kristina may care more about how the fields are named.

Reply:
Yea, I've definitely been a little bit more paranoid about the field names from the qa testing. I didn't really see the need for `dutyofcare_commons.dutyofcare*` in the db when revising the fields, but I'll talk w/ Kristina about it as well for the upcoming procedures. 

I was planning on asking her for the data qa testing on dev once the procedures go up as well since it might be overwhelming to do the 6 or 7 new procedures we're getting at once.